### ISSUE TITLE
Support configuring sort parameters for media

### ISSUE THREAD

**Issue by rfsbraz** [rfsbraz/deleterr#24]

**Comment by rfsbraz** [comment]
I'm implementing media sort based on the following parameters:

- *title*
- *size*
- *status*
- *release_year*
- *runtime*
- *added_date*
- *rating*

All parameters support both `asc` and `desc` modes, and configurations will be per library.

```yaml
libraries:
  - name: 'Movies'
    type: 'Radarr'
    sort:
      field: 'title'  # Can be 'title', 'added_date', etc.
      order: 'asc'  # Can be 'asc' or 'desc'
  - name: 'TV Shows'
    type: 'Sonarr'
    sort:
      field: 'added_date'
      order: 'desc'
```

**Comment by rfsbraz** [comment]
Closed by #28
